app/templates/my-profile-simple.html

```diff
@@ -0,0 +1,49 @@
+{% extends "__base__.html" %}
+
+{% block title %}{{ gettext('My Profile') }}{% endblock %}
+
+{% block content %}
+
+<h1>{{ gettext("Success!") }}</h1>
+
+<p>{{ gettext("Now please share some basic info about yourself, so others can know who you are.") }}</p>
+
+<form class="footer-fix" method="post">
+{{ form.csrf_token }}
+
+<div class="form-horizontal">
+
+{% macro input(name) -%}
+<div class="form-group">
+  {{ form[name].label(class="col-md-4 control-label") }}
+  <div class="col-md-4">
+  {{ form[name](class="input-md form-control",placeholder=form[name].description) }}
+  {% if form[name].errors %}
+    <ul class="error">
+    {% for error in form[name].errors %}
+    <li>{{ error }}</li>
+    {% endfor %}
+    </ul>
+  {% endif %}
+  </div>
+</div>
+{%- endmacro %}
```

Comment:
This code is duplicated from `my-profile.html`, but I'm ok with that for now since we'll be moving to claudio's new form styling soon anyways, and can properly refactor things then... Unless anyone disagrees with this approach.